libtsuba/src/tsuba.cpp

```diff
@@ -202,14 +203,28 @@ tsuba::ListAvailableViews(
     });
   }
 
-  if (views_found.empty()) {
-    return KATANA_ERROR(
-        tsuba::ErrorCode::NotFound, "no views found for that version");
+  if (!acceptable_version_found) {
+    if (version) {
+      return KATANA_ERROR(
+          tsuba::ErrorCode::NotFound,
+          "no views found for version {} in directory {}", version.value(),
+          rdg_dir);
+    } else {
+      return KATANA_ERROR(
+          tsuba::ErrorCode::NotFound,
+          "no manifest files found in directory: {}", rdg_dir);
```

Comment:
ditto here on the context.  In a bunch of other places it's up to the caller to add context when the context is one of the parameters the caller provided.